products/sle/main.pm

```diff
@@ -1005,14 +1005,17 @@ sub load_slenkins_tests {
 sub load_ha_cluster_tests {
     return unless (get_var("HA_CLUSTER"));
 
+    # We need to boot *before* waiting for support server
+    # Because of TianoCore UEFI boot method
+    loadtest "boot/boot_to_desktop";
```

Comment:
ok, it is clear

Reply:
Yes, I added a note to not forget in six months ;-)